container/container.go

```diff
@@ -68,6 +68,13 @@ type ExitStatus struct {
 	ExitedAt time.Time
 }
 
+// ConfigReference wraps swarmtypes.ConfigReference to add a Sensitive flag.
+type ConfigReference struct {
+	*swarmtypes.ConfigReference
+	// Sensitive is set if this config should not be written to disk.
+	Sensitive bool
```

Comment:
Silly question: should the templating driver also be aware of the content being sensitive or not? (just wondering if drivers may have to treat sensitive content different than non-sensitive)